python/pyspark/ml/classification.py

```diff
@@ -454,7 +454,8 @@ class TreeClassifierParams(object):
     impurity = Param(Params._dummy(), "impurity",
                      "Criterion used for information gain calculation (case-insensitive). " +
                      "Supported options: " +
-                     ", ".join(supportedImpurities), typeConverter=TypeConverters.toString)
+                     ", ".join(supportedImpurities) + ". (Default: gni)",
```

Comment:
gini not gni. Does the default have to be explicitly documented like this? I didn't see other params doing it but I just looked in this file